arisa.json

```diff
@@ -216,6 +216,76 @@
           "BDS",
           "MCD",
           "MCE"
+        ],
+        "fields": [
+          {
+            "fieldId": "customfield_10500",
+            "fieldName": "Confirmation Status",
+            "fieldType": "VALUE",
+            "defaultValue": "Unconfirmed",
+            "permissionGroups": [
+              "staff",
+              "global-moderators",
+              "helper"
+            ],
+            "message": "{panel:borderColor=orange}(!) Don't try to change \"Confirmation Status\". It may only be changed by moderators and helpers. Consider this a warning.{panel}"
+          },
+          {
+            "fieldId": "customfield_10701",
+            "fieldName": "CHK",
+            "fieldType": "STRING",
+            "defaultValue": null,
+            "permissionGroups": [
+              "staff",
+              "global-moderators",
+              "helper"
+            ],
+            "message": "{panel:borderColor=orange}(!) Don't try to change \"CHK\". It may only be changed by moderators and helpers. Consider this a warning.{panel}"
+          },
+          {
+            "fieldId": "customfield_11100",
+            "fieldName": "Linked",
+            "fieldType": "DOUBLE",
+            "defaultValue": null,
+            "permissionGroups": [
+              "staff",
+              "global-moderators",
+              "helper"
+            ],
+            "message": "{panel:borderColor=orange}(!) Don't try to change \"Linked\". It may only be changed by moderators and helpers. Consider this a warning.{panel}"
+          },
+          {
+            "fieldId": "customfield_10900",
+            "fieldName": "Pinned",
+            "fieldType": "DOUBLE",
+            "defaultValue": null,
+            "permissionGroups": [
+              "staff",
+              "global-moderators",
+              "helper"
+            ],
+            "message": "{panel:borderColor=orange}(!) Don't try to change \"Pinned\". It may only be changed by moderators and helpers. Consider this a warning.{panel}"
+          },
+          {
+            "fieldId": "customfield_12200",
+            "fieldName": "Mojang Priority",
+            "fieldType": "VALUE",
+            "defaultValue": null,
+            "permissionGroups": [
+              "staff"
+            ],
+            "message": "{panel:borderColor=orange}(!) Don't try to change \"Mojang Priority\". It may only be changed by the developers of the game. Consider this a warning.{panel}"
+          },
+          {
+            "fieldId": "customfield_12201",
+            "fieldName": "Triaged Time",
+            "fieldType": "STRING",
+            "defaultValue": null,
+            "permissionGroups": [
+              "staff"
+            ],
+            "message": "{panel:borderColor=orange}(!) Don't try to change \"Triaged Time\". It may only be changed by the developers of the game. Consider this a warning.{panel}"
```

Comment:
the "Consider this a warning" seems a bit direct 😅, can we use something similar to the the keep private message?

Reply:
This is based on `{panel:borderColor=orange}(!) Don't try to change "Mojang Priority". It may only be changed by the developers. Consider this a warning.{panel}` from `Panel - Mojang Priority`  from the helper messages.
Maybe it should be a little less harsh for bot messages or fields that only require helper?